www/pages/project-details.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+permalink: /project-details
+title: Project history and versions
+---
+
+In Mergin, you can see the details of the changes made to the project from different devices or users. Each time you sync the project from your mobile device or from QGIS through the plugin, a new version will be created.
+
+On the Mergin website, you can view what files have been added or removed. If you use Geopackage for your survey, you can also see the list the features which have been added, deleted or updated.
+
```

Comment:
It would be nice to add a screenshot of the page with a list of versions of a project (ideally with multiple authors)